public/global.json

```diff
@@ -51,5 +51,8 @@
         "demo-image": {
             "path": "demo-image/config.json"
         }
-    }
+    },
+    "adminUsers":[
+        "briancbollen@gmail.com"
+    ]

```

Comment:
Yeah, mistakenly added. Will fix that. I think by default though we'll have to have it blank because of the logic. Otherwise, it will try to add that email I believe and then you'll be locked out.